docs/checkit.js

```diff
@@ -793,38 +792,40 @@ CheckIt.prototype.getNumberOfChecked = function() {
     var checkit = this;
 
     return checkit.store.getActive().then(function(activeCalendarId) {
-        console.log("found this activeCalendarId: " + activeCalendarId);
-        return checkit.store.loadById(activeCalendarId).catch(function(err) {
-            console.log("Couldn't find active calendar specifically in getNumberOfChecked");
+        
+        
+        
+        
+        /*console.log("found this activeCalendarId: " + activeCalendarId);
+    return checkit.store.getActive()
+        .then(function(activeCalendarId) {
+            return checkit.store.loadById(activeCalendarId)
+        .catch(function(err) {
             console.log("There is no current active calendar " + err);
             checkit.store.removeActive();
+            checkit.showForm(checkit.$buildCalendarForm);
             return err;
             });
         })
         .then(function(activeCalendarState) {
-            
             if (activeCalendarState !== null) {
                 var state = activeCalendarState;
                 var calendar = new Calendar(state);
-                console.log("found calendar:");
-                console.log(calendar);
-                var size = 0, key;
-                for (key in calendar.state.checkedDays) {
-                    if (calendar.state.checkedDays.hasOwnProperty(key)) {
-                        size++;
-                    }
-                }
-                console.log('PRINTING SIZE');
-                console.log(size);
-                return size;
+                return checkit.displayCalendar(calendar);
             }
-            
         })
         .catch(function(err) {
-            console.log("getNumberOfChecked");
-            console.log("Could not get number of checked days" + err);
+            console.log("Could not display active calendar" + err);
+            checkit.showForm(checkit.$buildCalendarForm);
             return err;
```

Comment:
If the function is supposed to return a number and you can't then that's an error.